### PR TITLE
Software License NFTs

### PR DESCRIPTION
**Pull request by awantoch**

### CODE REVIEW

File: packages/evm-contracts/contracts/IValist.sol [hunk 1]
```diff
@@ -150,6 +162,16 @@ interface IValist {
   /// @param _address Address of member.
   function removeTeamMember(string memory _teamName, address _address) external;
 
+  /// Set team beneficiary address for recieving payments.
+  ///
+  /// @param _teamID Unique ID of the team.
+  /// @param _newBeneficiary Address of new beneficiary address.
+  function setBeneficiary(
```
Comment (nasdf) [inline packages/evm-contracts/contracts/IValist.sol:169]: `setTeamBeneficiary`would be more consistent.

File: packages/evm-contracts/contracts/IValist.sol [hunk 2]
```diff
@@ -6,49 +6,60 @@ interface IValist {
...
   /// @dev emitted when a new team member is added
-  event TeamMemberAdded(string _teamName, address _member);
+  event TeamMemberAdded(string _teamName, address _member, address _sender);
   /// @dev emitted when an existing team member is removed
-  event TeamMemberRemoved(string _teamName, address _member);
+  event TeamMemberRemoved(string _teamName, address _member, address _sender);
+
+  /// @dev emitted when a beneficiary is updated on a license.
+
+  event BeneficiaryUpdated(
```
Comment (nasdf) [inline packages/evm-contracts/contracts/IValist.sol:17]: `TeamBeneficiaryUpdated` would be more consistent

**Review by nasdf — APPROVED** [review]
I think we should have the functionality to limit licenses to a certain amount.  For instance if a game has a limited collectors editions.

Overall looks great!

File: packages/evm-contracts/contracts/IValist.sol (hunk 1 above)
Comment (awantoch) [reply packages/evm-contracts/contracts/IValist.sol]: updated!

File: packages/evm-contracts/contracts/IValist.sol (hunk 2 above)
Comment (awantoch) [reply packages/evm-contracts/contracts/IValist.sol]: updated!

**Comment by awantoch** [comment]
> I think we should have the functionality to limit licenses to a certain amount. For instance if a game has a limited collectors editions.
> 
> Overall looks great!

Totally agreed! Just added that to the TODO :D

**Pull request merged**